### PR TITLE
Add Patroni failsafe_mode parameter

### PR DESCRIPTION
**Pull request by hughcapet**
Add support for the [upcoming Patroni feature](https://github.com/zalando/patroni/pull/2379)

### CODE REVIEW

**Comment by FxKu** [comment]
:+1:

File: docs/reference/cluster_manifest.md [hunk 1]
```diff
@@ -316,6 +316,9 @@ explanation of `ttl` and `loop_wait` parameters.
 
 * **synchronous_node_count**
   Patroni `synchronous_node_count` parameter value. Note, this option is only available for Spilo images with Patroni 2.0+. The default is set to `1`. Optional.
+
+* **failsafe_mode**
+  Patroni `failsafe_mode` parameter value. Note, this option is currently not included in any Patroni release. The default is set to `false`. Optional.
```
Comment (FxKu) [inline docs/reference/cluster_manifest.md:321]: Maybe one more sentence what this mode is about :smiley: (DCS down etc.)

**Comment by FxKu** [comment]
:+1:

**Comment by idanovinda** [comment]
👍

**Pull request merged**